comp/core/internal/params.go

```diff
@@ -37,6 +37,14 @@ type BundleParams struct {
 	// the component's config data.
 	ConfigLoadSysProbe bool
 
+	// SecurityAgentConfigFilePaths are the paths at which to look for security-aegnt
+	// configuration, usually given by the --cfgpath command-line flag.
+	SecurityAgentConfigFilePaths []string
+
+	// ConfigLoadSecurityAgent determines whether to read the config from
+	// SecurityAgentConfigFilePaths or from ConfFilePath.
+	ConfigLoadSecurityAgent bool
```

Comment:
This reads to me as "use `ConfigFilePath string` or use `ConfigFilePaths []string`".  Could those be combined by just always accepting `[]string`?  Then agents with only one file can just use a one-element slice.

Reply:
I think this could be doable yes, this will just mean a bit more change in the PR because currently the security agent kind of ignore all other settings (like `withoutSecret` etc)

Reply:
Does that make more sense in a followup, or would that end up being more work?

I'm happy for component work to generate a healthy backlog of "refactor X to do Y" tickets.  Those are good rainy-afternoon tickets for later!

Reply:
I think it could be done in a follow up yes

Reply:
https://datadoghq.atlassian.net/browse/SEC-5785 to track the followup